docs/connect/java.md

```diff
@@ -2,11 +2,93 @@
 
 # Java
 
-JDBC is a standard Java API that provides a common interface for accessing
+:::{include} /_include/links.md
+:::
+
+:::{div} sd-text-muted
+Java applications mostly use JDBC to connect to CrateDB.
+:::
+
+:::{div}
+[JDBC] is a standard Java API that provides a common interface for accessing
 databases in Java.
+:::
+
+:::{rubric} Driver options
+:::
+
+:::{div}
+You have two JDBC driver options: The [PostgreSQL
+JDBC Driver] and the {ref}`crate-jdbc:index`.
+PostgreSQL JDBC uses the `jdbc:postgresql://` protocol identifier,
+while CrateDB JDBC uses `jdbc:crate://`.
+:::
+
+Applications using the PostgreSQL JDBC Driver may emit PostgreSQL-specific
```

Comment:
imho we should rephrase to encourage using the postgres jdbc and only if they encounter problems turn to our crate one.